testutil/ciao-down/workloads/ciao.yaml

```diff
@@ -227,6 +224,9 @@ runcmd:
  - {{endTaskCheck .}}
 {{end}}
 
+ - echo "export GOPATH={{template "GOPATH" . }}" >> /home/{{.User}}/.bashrc
+ - echo "export PATH=$PATH:{{template "GOPATH" . }}/bin:/usr/local/go/bin:/usr/local/nodejs/bin" >> /home/{{.User}}/.bashrc
```

Comment:
@markdryan putting gopath in /etc/environment may still be needed for the sudo invocation of go tests. The networking unit tests use sudo go test... . Worst case we need to ensure our tests give the full path to like
sudo `which go` test